src/string_immutable.zig

```diff
@@ -5905,20 +5903,29 @@ pub const visible = struct {
         return len;
     }
 
-    fn visibleUTF16WidthFn(input: []const u16, comptime asciiFn: anytype) usize {
-        var bytes = input;
+    fn visibleUTF16WidthFn(input: []const u16, exclude_ansi_colors: bool) usize {
         var len: usize = 0;
-        while (bun.strings.firstNonASCII16CheckMin([]const u16, bytes, false)) |i| {
-            len += asciiFn(bytes[0..i]);
-            bytes = bytes[i..];
-
-            const utf8 = utf16CodepointWithFFFD([]const u16, bytes);
-            len += visibleCodepointWidthType(u32, utf8.code_point);
-            bytes = bytes[@min(@as(usize, utf8.len), bytes.len)..];
+        var iter = std.unicode.Utf16LeIterator.init(input);
```

Comment:
1) This is going to be quite a lot slower than the previous implementation. `firstNonASCII16CheckMin` should do the right  thing, then if the u16 char is > 255 then it needs to be decoded as UTF-16. `min` here means is it >= minimum non human readable character
2) We shouldn't use `std.unicode.*` at runtime in Bun's codebase, as it is less tested in production than our own implementations of these things and makes assumptions which are too restrictive for us to depend on